doc/uftrace-tui.md

```diff
@@ -147,6 +147,7 @@ Following keys can be used in the TUI window:
  * `g`:                   Show backtrace and call graph of the current function
  * `R`:                   Show uftrace report
  * `r`:                   Show uftrace report of the current function
+ * `s`:                   Show uftrace sorted report by KEYs
```

Comment:
Please reorder as follows.
```suggestion
 * `s`:                   Show uftrace report sorted by KEYs
```
In addition, you can also add same description in `doc/ko/uftrace-tui.md` man page in Korean.

Reply:
I fixed it as you told me. Please check.